packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -35,3 +35,9 @@ dev_dependencies:
   pedantic: ^1.10.0
   plugin_platform_interface: ^2.0.0
   stream_transform: ^2.0.0
+
+# TODO: Remove ovveride before merge
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides:
+  google_maps_flutter_platform_interface:
+    path: ../google_maps_flutter_platform_interface
```

Comment:
All these overrides make this PR unpublishable, once the approach is approved this needs to be split into one PR per affected package.

Reply:
(This is just following our official guidance about multip-package federated reviews; see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins)